assets/styles/styles.css

```diff
@@ -0,0 +1,308 @@
+  /*max-width: 105rem;
+  width: 100%;*/
+}
+
+.header {
+  display: flex;
+  justify-content: center;
+
+  .container {
+    max-width: 132.2rem;
+    width: 100%;
+    height: 10.4rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 3.9rem;
+
+    .header__logo {
+      margin-right: auto;
+      text-decoration: none;
+      color: white;
+      font-size: 2.4rem;
+      font-weight: 700;
+      padding-left: 13.6rem;
+    }
+
+    .header__nav .nav__list {
+      display: flex;
+      align-items: center;
+      list-style: none;
+      padding: 0;
+      gap: 2.1rem;
+
+      .nav__link {
+        text-decoration: none;
+        color: white;
+        font-size: 1.4rem;
+        font-weight: 600;
+      }
+      .nav__link:hover {
+        text-decoration: underline;
+      }
+    }
+
+    .header__actions {
+      display: flex;
+      gap: 1.8rem;
+      padding-right: 16.4rem;
+
+      .action__btn {
+        all: unset;
+        cursor: pointer;
+        width: 1.6rem;
+        height: 1.6rem;
+      }
+    }
+  }
+}
+
+.main {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+
+  .container {
+    max-width: 1050px;
+    height: 571px;
+
+    .hero-image {
+      position: absolute;
+      top: 0;
+      left: 0;
+      z-index: -1;
+      width: 100%;
+      filter: brightness(50%);
+    }
+
+    .container1 {
+      display: flex;
+      max-width: 573px;
+      width: 100%;
+      padding: 120px 0;
+      flex-direction: column;
+      justify-content: center;
+      text-align: center;
+      align-items: center;
+      gap: 4rem;
+
+      .container1__h1 {
+        color: #ffffff;
+        font-size: 5.8rem;
+        font-weight: 800;
+        padding: 0;
+        margin: 0;
+      }
+
+      .container1__h4 {
+        color: #ffffff;
+        font-size: 2rem;
+        font-weight: 500;
+        padding: 0;
+        margin: 0;
+        display: flex;
+        justify-content: center;
+        text-align: center;
+        align-items: center;
+      }
+      .hero__actions {
+        display: flex;
+        gap: 1rem;
+        width: 34.9rem;
+        height: 4.8rem;
+        margin: 0;
+        padding: 0;
+
+        .hero__btn1 {
+          margin: 0;
+          padding: 0;
+          width: 18.5rem;
+          border-radius: 3.7rem;
+          border: 0;
+          background-color: #ffa62b;
+          font-size: 1.4rem;
+          font-weight: 700;
+          color: #ffffff;
+        }
+        .hero__btn2 {
+          margin: 0;
+          padding: 0;
+          width: 15.4rem;
+          border-radius: 3.7rem;
+          border: 0.1rem solid #ffffff;
+          background: transparent;
+          font-size: 1.4rem;
+          font-weight: 700;
+          color: #ffffff;
+        }
+      }
+    }
+  }
+}
+
+.footer {
+  display: flex;
+  justify-content: center;
+
+  .container {
+    max-width: 144rem;
+    width: 100%;
+    margin: 0;
```

Comment:
no need to reset the margin and padding, as the div doesn't have them by default.